core/templates/dev/head/services/ImprovementCardServiceSpec.ts

```diff
@@ -68,7 +89,9 @@ describe('ImprovementCardService', function() {
           done.fail(error);
         };
 
-        this.ImprovementCardService.fetchCards().then(onSuccess, onFailure);
+        ImprovementCardService.fetchCards().then(onSuccess, onFailure);
+
+        $rootScope.$apply();
```

Comment:
Why add this? Maybe add a comment to explain.

Reply:
Done.